WordPress/twentytwentytwo
# Add missing comments block to page template

**Description**

Twenty Twenty-Two must support comments on Page post type, if enabled on a specific page using the "discussion" settings:
<img width="1111" alt="Capture d’écran 2021-10-18 à 10 26 39" src="https://user-images.githubusercontent.com/1590998/137697490-59e8e8b1-245a-4c25-a805-8ed0ee48887d.png">

We simply need to add the comment block template to the page.html template.

**Screenshots**

Before the PR: a Page with comment option enabled didn't show any comment/comment form.
<img width="884" alt="Capture d’écran 2021-10-18 à 10 27 02" src="https://user-images.githubusercontent.com/1590998/137697909-ca6527f1-a45c-42d8-bce6-6b998f592813.png">


After the PR: comments/comment form is displayed when the option is enabled.
<img width="917" alt="Capture d’écran 2021-10-18 à 10 35 52" src="https://user-images.githubusercontent.com/1590998/137697953-0e0a9881-ecb4-41a0-b99b-b9246688ab6c.png">


**Testing Instructions** 

1. Enable comments on a Page using the "Discussion" settings of this specific page.
2. On front-end, check wether the comments list/comment form is displayed.
